compiler/test/suites/flags.re

```diff
@@ -0,0 +1,85 @@
+open Grain_tests.TestFramework;
+open Grain_tests.Runner;
+open Grain_codegen;
+open Binaryen;
+
+let load_module = fullpath => {
+  let ic = open_in_bin(fullpath);
+  let length = in_channel_length(ic);
+  let module_bytes = Bytes.create(length);
+  really_input(ic, module_bytes, 0, length);
+  close_in(ic);
+  Module.read(module_bytes);
+};
+
+describe("flags", ({test, testSkip}) => {
+  // Test --no-bulk-memory polyfill
+  test("bulk_memory_calls", ({expect}) => {
+    let name = "bulk_memory_calls";
+    let outfile = wasmfile(name);
+    ignore @@
+    compile(
+      ~link=true,
+      ~config_fn=() => {Grain_utils.Config.bulk_memory := true},
+      name,
+      {|
+      module BulkMemoryCalls
+
+      from "runtime/unsafe/memory" include Memory
+      @disableGC
+      provide let foo = () => {
+        Memory.fill(0n, 0n, 0n)
+        Memory.copy(0n, 0n, 0n)
+      }
+      |},
+    );
+    let wasm_mod = load_module(outfile);
+    Module.set_features(wasm_mod, Compcore.features);
+    expect.bool(
+      List.mem(Module.Feature.bulk_memory, Module.get_features(wasm_mod)),
+    ).
+      toBe(
+      true,
+    );
+    // Validate the module uses the bulk memory feature
+    expect.int(Module.validate(wasm_mod)).toBe(1);
```

Comment:
This is just checking that Binaryen can get and set features. I don't think we need this test.